src/components/Sidebar/SidebarIcon.tsx

```diff
@@ -0,0 +1,59 @@
+export default function SidebarIcon({ label }: { label: string }) {
+    switch (label) {
```

Comment:
I try to avoid using switch statements. They tend to add unnecessary complexity. For this component, I think it would be more readable to make separate icon component files for each icon.